EIPS/eip-2771.md

```diff
@@ -0,0 +1,129 @@
+---
+eip: 2771
+title: Secure Protocol for Native Meta Transactions
+author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri),  Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Tomar Sachin (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman) 
+discussions-to: https://github.com/ethereum/EIPs/pull/2771
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-07-01
+requires: 2770
```

Comment:
Why does this require 2770?  I don't see any references to 2770 anywhere else in this EIP.

Reply:
Also, we can't merge this until #2770 is merged, so will have to wait a bit.

Reply:
The dependency on 2770 was a mistake. Removed!